src/utils/csp.js

```diff
@@ -0,0 +1,36 @@
+function buildPolicies(nonce) {
+  const policies = {
+    default: ["'self'"],
+    img: ["'self'", "www.google-analytics.com", "*.doubleclick.net"],
+    style: ["'self'", "fonts.googleapis.com"],
+    frame: ["www.google.com"],
+    font: ["fonts.googleapis.com", "fonts.gstatic.com"],
+    script: [
+      "www.googletagmanager.com",
+      "www.google-analytics.com",
+      "*.google.com",
+      `'nonce-${nonce}'`,
```

Comment:
Can we use 'self' here? Maybe we don't need nonces at all?

Reply:
will remove